.github/workflows/remove-wheels.yml

```diff
@@ -0,0 +1,81 @@
+name: Remove old wheels
+
+on:
+  # Run daily at 1:23 UTC
+  schedule:
+  - cron:  '23 1 * * *'
+  workflow_dispatch:
```

Comment:
Thanks for adding this in: useful to be able to launch with the click of a button!